Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/Visuals/Configuration/LinksConfiguration.cs

```diff
@@ -51,7 +51,7 @@ public float WireframeEdgeRadius
                 if (wireframeEdgeRadius != value)
                 {
                     wireframeEdgeRadius = value;
-                    configurationChanged.Invoke();
+                    wireFrameChanged.Invoke(WireframeChangedEventType.Radius);
```

Comment:
Do these events need `null` checks, or are they guaranteed to be _something_?

Reply:
No they shouldn't need a check - i'm initializing this event on declaration so afaik there's no way this class can live without the event being initialized